front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -792,6 +791,13 @@ export default function AppChat({
     })();
   };
 
+  function filterErrorMessages(m: ChatMessageType[]) {
```

Comment:
The effect won't last if the discussion is continued right? Maybe we want that even if seeing a new message answered will mitigate the effect we're trying to avoid here.

Still I would go for: Ignore error messages as well as the user message preceding them?